### PR TITLE
Feature/6 kiss3d scene

### LINKED ISSUE #6: Setup Kiss3D to render generic shapes

### PR DESCRIPTION
**Pull request by M-Blomqvist**
Closes #6 #27

### CODE REVIEW

**Comment by M-Blomqvist** [comment]
Also added an as_cube()

File: pphysx3d/src/renderer/mod.rs [hunk 1]
```diff
@@ -1 +1,128 @@
+use std::usize;
```
Comment (coffecup25) [inline pphysx3d/src/renderer/mod.rs:1]: Is this import really needed?

File: pphysx3d/src/renderer/mod.rs [hunk 2]
```diff
@@ -1 +1,128 @@
...
+            for go in gameobjects {
+                self.renderables[i].set_local_transformation(go.position);
+                i += 1;
+            }
+            return Ok(());
+        }
+        Err(String::from("Error: Window closed for rendering"))
+    }
+
+    /// Add a rendered SceneNide to the renderer
```
Comment (coffecup25) [inline pphysx3d/src/renderer/mod.rs:115]:
```suggestion
    /// Add a rendered SceneNode to the renderer
```

File: pphysx3d/src/renderer/mod.rs (hunk 1 above)
Comment (M-Blomqvist) [reply pphysx3d/src/renderer/mod.rs:1]: No, thats rust analyzer being derpy
```suggestion
```

**Review by coffecup25 — APPROVED** [review]
harasho

**Review by axellervik — APPROVED** [review]

**Pull request merged**